packages/bot-web-ui/src/stores/google-drive-store.js

```diff
@@ -29,55 +31,42 @@ export default class GoogleDriveStore {
         this.api_key = api;
     };
 
-    initialise() {
-        gapi.load('client:auth2:picker', {
-            callback: () => {
-                gapi.client
-                    .init({
-                        apiKey: this.api_key,
-                        clientId: this.client_id,
-                        scope: 'https://www.googleapis.com/auth/drive.file',
-                        discoveryDocs: ['https://www.googleapis.com/discovery/v1/apis/drive/v3/rest'],
-                    })
-                    .then(
-                        () => {
-                            this.google_auth = gapi.auth2.getAuthInstance();
-                            this.google_auth.isSignedIn.listen(is_signed_in => this.updateSigninStatus(is_signed_in));
-                            this.updateSigninStatus(this.google_auth.isSignedIn.get());
-                        },
-                        error => {
-                            // TODO
-                            console.warn(error); // eslint-disable-line
-                        }
-                    );
+    initialise = () => {
+        gapi.load('client:picker', () =>
+            gapi.client.load('https://www.googleapis.com/discovery/v1/apis/drive/v3/rest')
+        );
+    };
+
+    initialiseClient = () => {
+        this.client = google.accounts.oauth2.initTokenClient({
+            client_id: this.client_id,
+            scope: 'https://www.googleapis.com/auth/drive.file',
```

Comment:
add a constant for this url name it SCOPE and use it

Reply:
done.